src/server/auth/server/api_server.go

```diff
@@ -214,6 +204,15 @@ func NewAuthServer(
 		// Watch for changes to the cluster role binding
 		go s.clusterRoleBindingCache.Watch()
 	}
+
+	// purge expired tokens every 24 hours
+	go func(a *apiServer, intervalSeconds int) {
```

Comment:
This is mostly just personal preference, but it feels more idiomatic to have this be a method attached to the apiServer struct, like `func (a *apiServer) purgeExpiredTokensRoutine()`. `intervalSeconds` could also be pulled out as a constant `time.Duration` at the top of this file.